### PR TITLE
Fix candle manager max candles

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Commons/pull/17
requires https://github.com/Drakkar-Software/OctoBot-Commons/pull/18

### CODE REVIEW

File: octobot_trading/data_manager/candles_manager.py [hunk 1]
```diff
@@ -117,64 +118,62 @@ def add_old_and_new_candles(self, candles_data):
         except IndexError as e:
             self.logger.error(f"Fail to add last candle {candles_data} : {e}")
 
-    def add_new_candle(self, new_candle_data: Dict):
-        if self.__should_add_new_candle(new_candle_data[PriceIndexes.IND_PRICE_TIME.value]):
+    def add_new_candle(self, new_candle_data: list):
```
Comment (Herklos) [inline octobot_trading/data_manager/candles_manager.py]: :+1:

File: tests/data_manager/test_candles_manager.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
+#  Drakkar-Software OctoBot
```
Comment (Herklos) [inline tests/data_manager/test_candles_manager.py]: :100:

File: .travis.yml [hunk 3]
```diff
@@ -14,11 +14,8 @@ env:
...
     - secure: wK6A14jETV7tfcM0BCW/LVhtIFELvwErVGFKuMSpfA6X20SIHxRUZx0GbAomi9VJIH/M/GDcxoQvduf6ciwRF6s1QLJJQZ2fSkK5rSeOXLe9IXY9qLF09c+MHq7VNxzsQjChxuwQt2Kl/luu1irflb1tnKmfqzg2aZve+3b7G05QwPmiHRJh518RTu8QHh2SbgrBUo34tg+2peVu27AEUxmkZS9fPHKLDqNd/U4tGxeTfy+fxVsCQwyKu0eRCFkCoWeQkT77WeXqippsrFFYPaOb9JpN4ThDvGOnc+egt4hfV4HUG7MffwcLhoNEHrcUZQYhB3Jgx+drUYVKk7HwF8eZrPTXCT4CMQALR3pknwosj1bvA2YnWJaMeXwb+wS2y4YFqCj4PrdEZoAfcd1Y2SQeJDKFpWxtVjZ2F/wWqarIO7hsIbNESPkui3WXbA3UXoHAQRZgAkQqfeu7BQBGdxXSvrUDiTYqidlkgHrQu+R6yc88XFd6iugRgUu+YTFX0577vUcaJpadSzeAnDJSwwS8e9gNZQcONAG0t/1lTQVwUnFdYGsqRWPaWC12xXoyLSm9B8RlN1+j4xmFZNJq55Jz2FsLCvwke0HJ1o6ppIjqhBT5kqLUqbzjEGJzIXCLUrFdktEb73VgQJCQmiz0aTs1cReninPQR06BP/0pbXc=
 
-before_install:
-    - sudo apt-get -y install liblapack-dev libblas-dev gfortran # fix scipy liblas error
-
 install:
-  - python3 -m pip install --prefer-binary --only-binary=scipy -r dev_requirements.txt -r requirements.txt
```
Comment (Herklos) [inline .travis.yml]: Why ?

Comment (GuillaumeDSM) [reply .travis.yml]: scipy is not required anymore since it was only used to shift numpy arrays

**Pull request merged**